tests/phpunit/api/v3/EmailTest.php

```diff
@@ -448,7 +474,25 @@ public function testEmailOnHold() {
     $this->assertEquals(date('Y-m-d H:i'), date('Y-m-d H:i', strtotime($result_change['values'][$result_change['id']]['reset_date'])));
     $this->assertEmpty($result_change['values'][$result_change['id']]['hold_date']);
 
-    $delresult = $this->callAPISuccess('email', 'delete', ['id' => $result['id']]);
+    $this->callAPISuccess('email', 'delete', ['id' => $result['id']]);
+  }
+
+  /**
+   * Test setting a bulk email unsets others on the contact.
+   *
+   * @throws \CRM_Core_Exception
+   */
+  public function testSetBulkEmail() {
```

Comment:
@eileenmcnaughton what if you set the multiple billing settings so you end up with multiple bulk mailing emails or whatever it is?

Reply:
@seamuslee001 I didn't aim to cover all the scenarios - I think this is enough additional test cover for this change